arrow/src/main/java/org/apache/calcite/adapter/arrow/ArrowDataTypeConstants.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.arrow;
+
+/**
+ * Constants for Arrow data types.
+ */
+public interface ArrowDataTypeConstants {
+  int ARROW_DEFAULT_DECIMAL_PRECISION = 19;
```

Comment:
Is the decimal precision an ARROW default value or a user-defined variable? As far as I know, when the RelNode is in the Adapter, we should already have a precision and Scale in Decimal Type.

Reply:
You are right, the decimal of arrow must specify the precision